src/lib/components/input/toggle/index.stories.tsx

```diff
@@ -0,0 +1,191 @@
+
+import { useState } from 'react';
+import { Toggle, ToggleProps } from '.';
+
+const story = {
+  title: 'JSX/Inputs/Toggle',
+  component: Toggle,
+  argTypes: {
+    options: {
+      description: 'Object that contains the possible options for rendering in the input. To create an icon-only toggle, pass an empty string.',
+      defaultValue: {
+        CAT:{
+          title: 'Cat',
+          description: 'At least 1'
+        },
+        DOG:{
+          title: 'Dog',
+          description: 'At least 2'
+        },
+        NONE:{
+          title: 'None',
+          description: 'No pets'
+        }
+      }
+    },
+    value: {
+      description: 'Current checked values.',
+    },
+    onChange: {
+      description: 'Function called everytime a value changes.',
+      action: true,
+      table: {
+        category: "Callbacks",
+      },
+    },
+    bordered: {
+      control: { type: 'boolean' },
+      description: 'Property that defines if checkbox should show the border around each label',
+      defaultValue: true
+    },
+    inlineLayout: {
+      description: 'Property that defines if options should show inline instead of block. Check inline checkbox options story for examples.',
+      defaultValue: false
+    },
+    className: {
```

Comment:
Shouldn't this be `classNames` to match the prop?

```suggestion
    classNames: {
```